app/models/error_detail.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ErrorDetail < ApplicationRecord
+  include Discard::Model
+  self.discard_column = :deleted_at
+
+  belongs_to :integration, polymorphic: true, optional: true
+  belongs_to :owner, polymorphic: true
+  belongs_to :organization
+
+  ERROR_CODES = %w[not_provided]
```

Comment:
So the goal is to extend this list with all possible error codes that will be used?

Reply:
yes 😁